cli/src/potpie/api_wrapper.py

```
@@ -0,0 +1,210 @@
+
+                    if response.status != 200:
+                        error_text = await response.text()
+                        logging.error(f"Failed to interact with agent: {error_text}")
+                        raise Exception("Failed to interact with agent.")
+
+                    async for line in response.content.iter_chunks():
+                        try:
+                            json_string = line[0].decode()
+                            data = json.loads(json_string)
+                            yield data["message"]
+                        except json.JSONDecodeError as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)